src/mixins/date-mixin.ts

```diff
@@ -101,13 +103,27 @@ export default class DateMixin extends Vue {
 
     // fix for Jest (which outputs MM/DD/YYYY no matter which 'en' locale is used)
     if (localTime.indexOf('/') >= 0) {
-      let date = localTime.substr(0, 10)
+      const date = localTime.substr(0, 10).split('/')
       const time = localTime.slice(11)
-      const dateParts = date.split('/')
-      date = `${dateParts[2]}-${dateParts[0]}-${dateParts[1]}`
-      localTime = `${date} ${time}`
+      // set as YYYY-MM-DD HH:MM:SS AM/PM
+      localTime = `${date[2]}-${date[0]}-${date[1]} ${time}`
     }
 
     return localTime
   }
+
+  /**
+   * The number of days that 'date' is from today.
+   * @returns -1 for yesterday
+   * @returns 0 for today
+   * @returns +1 for tomorrow
+   * @returns NaN in case of error
+   */
+  daysFromToday (date: string): number {
+    if (!date) return NaN
+    // calculate difference between start of "today" and start of "date" (in local time)
+    const todayLocalMs = new Date().setHours(0, 0, 0, 0)
+    const dateLocalMs = new Date(date).setHours(0, 0, 0, 0)
+    return Math.round((dateLocalMs - todayLocalMs) / this.MS_IN_A_DAY)
+  }
```

Comment:
What is the application of this method?

Reply:
```
    expiresText (task): string {
      const expireDays = this.daysFromToday(task.nameRequest?.expirationDate)
      if (isNaN(expireDays) || expireDays < 0) {
        return 'Expired'
      } else if (expireDays < 1) {
        return 'Expires today'
      } else if (expireDays < 2) {
        return 'Expires tomorrow'
      } else {
        return `Expires in ${expireDays} days`
      }
    },
```

Reply:
Basically return 0 if date is today, -X if date is X days in the past, or +X if date is X days in the future.